docs/conf.py

```diff
@@ -7,9 +7,9 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = 'buildingSMART Validation Service'
-copyright = '2024, buildingSMART International'
+copyright = '2025, buildingSMART International'
 author = 'buildingSMART International'
-release = '0.6'
+release = '0.7.7'
```

Comment:
Added a note in the release management docs to update this version with every deployment to prod